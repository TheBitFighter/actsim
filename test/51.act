defproc test()
{
  int<72> x;

  chp {
    x := 1;
   *[ x{71} = 0 ->  log ("%xiter:", x); x := x << 1 ]
  }
}
