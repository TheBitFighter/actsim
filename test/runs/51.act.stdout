[                  10] <>  iter:00000000000000001
[                  20] <>  iter:00000000000000002
[                  30] <>  iter:00000000000000004
[                  40] <>  iter:00000000000000008
[                  50] <>  iter:00000000000000010
[                  60] <>  iter:00000000000000020
[                  70] <>  iter:00000000000000040
[                  80] <>  iter:00000000000000080
[                  90] <>  iter:00000000000000100
[                 100] <>  iter:00000000000000200
[                 110] <>  iter:00000000000000400
[                 120] <>  iter:00000000000000800
[                 130] <>  iter:00000000000001000
[                 140] <>  iter:00000000000002000
[                 150] <>  iter:00000000000004000
[                 160] <>  iter:00000000000008000
[                 170] <>  iter:00000000000010000
[                 180] <>  iter:00000000000020000
[                 190] <>  iter:00000000000040000
[                 200] <>  iter:00000000000080000
[                 210] <>  iter:00000000000100000
[                 220] <>  iter:00000000000200000
[                 230] <>  iter:00000000000400000
[                 240] <>  iter:00000000000800000
[                 250] <>  iter:00000000001000000
[                 260] <>  iter:00000000002000000
[                 270] <>  iter:00000000004000000
[                 280] <>  iter:00000000008000000
[                 290] <>  iter:00000000010000000
[                 300] <>  iter:00000000020000000
[                 310] <>  iter:00000000040000000
[                 320] <>  iter:00000000080000000
[                 330] <>  iter:00000000100000000
[                 340] <>  iter:00000000200000000
[                 350] <>  iter:00000000400000000
[                 360] <>  iter:00000000800000000
[                 370] <>  iter:00000001000000000
[                 380] <>  iter:00000002000000000
[                 390] <>  iter:00000004000000000
[                 400] <>  iter:00000008000000000
[                 410] <>  iter:00000010000000000
[                 420] <>  iter:00000020000000000
[                 430] <>  iter:00000040000000000
[                 440] <>  iter:00000080000000000
[                 450] <>  iter:00000100000000000
[                 460] <>  iter:00000200000000000
[                 470] <>  iter:00000400000000000
[                 480] <>  iter:00000800000000000
[                 490] <>  iter:00001000000000000
[                 500] <>  iter:00002000000000000
[                 510] <>  iter:00004000000000000
[                 520] <>  iter:00008000000000000
[                 530] <>  iter:00010000000000000
[                 540] <>  iter:00020000000000000
[                 550] <>  iter:00040000000000000
[                 560] <>  iter:00080000000000000
[                 570] <>  iter:00100000000000000
[                 580] <>  iter:00200000000000000
[                 590] <>  iter:00400000000000000
[                 600] <>  iter:00800000000000000
[                 610] <>  iter:01000000000000000
[                 620] <>  iter:02000000000000000
[                 630] <>  iter:04000000000000000
[                 640] <>  iter:08000000000000000
[                 650] <>  iter:10000000000000000
[                 660] <>  iter:20000000000000000
[                 670] <>  iter:40000000000000000
[                 680] <>  iter:80000000000000000
[                 690] <>  iter:100000000000000000
[                 700] <>  iter:200000000000000000
[                 710] <>  iter:400000000000000000
